src/balancer/accept_http.rs

```diff
@@ -425,14 +419,15 @@ pub async fn accept_request(
     //
     // Also handle cache insertions.
     let time = Instant::now();
-    (response, rpc_position) = forward_body(
+    let (response, rpc_position) = forward_body(
```

Comment:
this is fine for how it is now, but it would probably be a good idea to refactor this so you only pass in `tx`, `connection_params` and `params. looks pretty messy now